plugins/ssl/ssl-certificate-expiry

```diff
@@ -102,26 +106,34 @@ parse_valid_days_from_certificate() {
 print_expire_days() {
     local host="$1"
     local port="$2"
+    local starttls="$3"
 
     # Wrap IPv6 addresses in square brackets
     echo "$host" | grep -q ':' && host="[$host]"
 
-    echo "" | openssl s_client -CApath /etc/ssl/certs \
+    if [ -n "$starttls" ]; then
+        echo "" | openssl s_client -CApath /etc/ssl/certs \
```

Comment:
I would suggest to add a parameter to the openssl call instead of duplicating the line.

Reply:
openssl "-starttls" argument cannot be empty, so the call MUST be different.

Reply:
I was a bit too concise, I guess :)

I imagined something like this:
```shell
local s_client_args=
[ -n "$starttls" ] && s_client_args="-starttls $starttls"
```
Afterwards I would add the unquoted `$s_client_args` parameter to the `s_client` call. This approach requires a `shellcheck` warning needs to be silenced (`# shellcheck disable=SC2086`).

What do you think?

Reply:
Good idea, though I don't know what shellcheck is...
I updated the PR with your improvement.